boards/risc-v/esp32c6/esp32c6-devkitm/configs/spi/defconfig

```diff
@@ -23,6 +23,7 @@ CONFIG_BUILTIN=y
 CONFIG_DEV_ZERO=y
 CONFIG_ESPRESSIF_ESP32C6=y
 CONFIG_ESPRESSIF_SPI2=y
+CONFIG_ESPRESSIF_SPI2_CSPIN=10
```

Comment:
Why CSPIN is = 10 if it isn't exposed in dev kit?

Reply:
CS pin for C6 is also main UART TX pin. For that reason it needs to be changed